source/tutorials/pd-mice.rst

```diff
@@ -627,47 +627,59 @@ Before we begin, we will filter out low abundance/low prevalence ASVs. Filtering
      --p-min-samples 4 \
      --o-filtered-table ./table_2k_abund.qza
 
-ANCOM fundamentally operates on a ``FeatureTable[Frequency]``, which contains the frequencies of features in each sample. However, ANCOM cannot tolerate zeros because compositional methods typically use a log-transform or a ratio and you can't take the log or divide by zeros. To remove the zeros from our table, we can add a pseudocount to the ``FeatureTable[Frequency]`` Artifact, creating a ``FeatureTable[Composition]`` in its place.
+ANCOM-BC fundamentally operates on a FeatureTable[Frequency], which contains the frequencies of features in each sample. ANCOM-BC accounts for sampling fraction by introducing a sample-specific offset term in a linear regression framework that is estimated from the observed data. The offset term serves as the bias correction, and the linear regression framework in log scale is analogous to log-ratio transformation to deal with the compositionality of microbiome data.
 
-.. command-block::
+Let’s use ANCOM-BC to check whether there is a difference in the gut microbiome of the mice based on their donor and their genetic background. We'll set a significance threshold for each group to narrow down the features based on their log-fold change (LFC) value. Features above that threshold will not be included in each group's visualization.
```

Comment:
It looks like your threshold is based on the q-value and not the log fold change value. Is that correct?

Reply:
yep, good catch - thank you!